src/rust/ide/lib/view/src/graph_editor.rs

```diff
@@ -1092,17 +1105,30 @@ impl GraphEditorModel {
         let logger         = Logger::new("GraphEditor");
         let display_object = display::object::Instance::new(&logger);
         let nodes          = Nodes::new(&logger);
-//        let visualizations = Stage::new(scene.clone_ref(), Logger::new("VisualisationCollection"));
+//      let visualizations = Stage::new(scene.clone_ref(), Logger::new("VisualisationCollection"));
+        let doc_view       = documentation::View::new(&scene);
         let edges          = default();
         let frp            = FrpInputs::new(network);
         let touch_state    = TouchState::new(network,&scene.mouse.frp);
         let project_name   = component::ProjectName::new(scene,focus_manager);
         display_object.add_child(&project_name);
-        let screen = scene.camera().screen();
-        let margin = 10.0;
-        project_name.set_position(Vector3::new(0.0,screen.height / 2.0 - margin,0.0));
+        let screen     = scene.camera().screen();
+        let margin     = 10.0;
+        project_name.set_position_y(screen.height / 2.0 - margin);
+        let doc_width  = documentation::DOC_VIEW_WIDTH;
+        let doc_margin = documentation::DOC_VIEW_MARGIN;
+        frp::extend! { network
+            eval scene.frp.shape((shape) {
+                doc_view.set_position_x((shape.width - doc_width) / 2.0 - doc_margin);
+                doc_view.frp.set_size.emit(Vector2::new(doc_width,shape.height - (doc_margin * 2.0)));
+            });
+        }
```

Comment:
this is doc initialization code. It sohuld not be placed in the iniialization code of graph editor. It should be placed inside of the initialization of the documentation view. I was writing about it in previous review.

Reply:
done